File: Java/Node/src/Node/Biz/Interfaces/GetStatus/IPostProcess.java
package Node.Biz.Interfaces.GetStatus;

import java.rmi.RemoteException;

import Node.Biz.Custom.PostParam;
/**
 * <p>This class create GetStatus Post Process Interface.</p>
 * <p>Company: enfoTech & Consulting, Inc.</p>
 * @author enfoTech
 * @version 2.0
 */

public interface IPostProcess {
	/**
	 * Execute
	 * @param token The authentication token
	 * @param transId The transaction ID
	 * @param param PostParam Object
	 * @throws RemoteException
	 * @return PostParam object
	 */
  public PostParam Execute (String token, String transID, PostParam param) throws RemoteException;
}

File: DotNet/Node.Core/Biz/Interfaces/GetStatus/IPostProcess.cs
using System;

using Node.Core.Biz.Manageable.Parameters;

namespace Node.Core.Biz.Interfaces.GetStatus
{
    /// <summary>
    /// The Interface for GetStatus PostProcess
    /// </summary>
    public interface IPostProcess
    {
        /// <summary>
        /// PostProcess for GetStatus.
        /// </summary>
        /// <param name="token">The Token which is result of Authentication Operation.</param>
        /// <param name="transID">The transaction id is result of Solicict/Submiet Operation.</param>
        /// <param name="param">Operation related parameters</param>
        /// <returns>Status String</returns>
        void Execute(string token, string transID, PostParam param);
    }
}
